src/orderbook.ts

```diff
@@ -83,6 +83,26 @@ export class Orderbook {
     });
   }
 
+  /**
+   * @param amount the amount of base tokens to be sold
+   * @return the price for which there are enough bids to fill the specified amount or undefined if there is not enough liquidity
+   */
+  priceToSellBaseToken(amount: number | BN) {
```

Comment:
Yes, it's just nicer for testing/demo purposes to be able to use readable numbers instead of having to always type `new BN()`. But in the practical use case we will be using it with BN instead of number.